tcollector.py

```diff
@@ -40,7 +40,7 @@
 from Queue import Queue
 from Queue import Empty
 from Queue import Full
-from optparse import OptionParser
+from optparse import OptionParser, SUPPRESS_HELP
```

Comment:
Unexpected error: <type 'exceptions.SystemExit'> error needs to be fixed on --help

Reply:
Nothing to fix here as --help always throws this error as it tries to look for all of the other registered command line arguments too.